docs/DeveloperGuide.md

```diff
@@ -322,6 +336,20 @@ The diagram below shows a sample interaction of `TaskDoneCommand`.
 
 ![TaskDoneSequenceDiagram](images/TaskDoneSequenceDiagram.png)    
 
+**TaskDelete command**
+ 
+The TaskDelete command allows the user to delete a task from a tag. 
+
+1. The command is passed into `LogicManager`.
+2. `LogicManager` calls the parseCommand method of `ProjactParser`.
+3. `ProjactParser` identifies the commandWord, which in this case is `taskdone` and the arguments.
+4. `ProjactParser` calls the parse method of `TaskDeleteCommandParser`, which parses the argument, cand returns a new `TaskDeleteCommand`.
+5. The `LogicManager` then calls the execute method of the `TaskDeleteCommand`, which calls the setTag method of `Model`.
+
+The diagram below shows a sample interaction of `TaskDeleteCommand`. 
+
```

Comment:
do you want to include why you implemented it this way? e.g. alphabetical index

Reply:
Ive edited the changes as this comment affects TaskDone as well!